gapic-generator-ads/docker-entrypoint.sh

```diff
@@ -26,10 +26,24 @@ while true; do
 done
 
 mkdir -p /workspace/out/lib
-exec grpc_tools_ruby_protoc \
-        --proto_path=/workspace/common-protos/ --proto_path=/workspace/in/ \
-        --ruby_out=/workspace/out/lib \
-        --grpc_out=/workspace/out/lib \
-        --ruby_ads_out=/workspace/out/ \
-        --ruby_ads_opt="configuration=/workspace/config.yml" \
-        `find /workspace/in/ -name *.proto`
+grpc_tools_ruby_protoc \
+  --proto_path=/workspace/common-protos/ --proto_path=/workspace/in/ \
+  --ruby_out=/workspace/out/lib \
+  --grpc_out=/workspace/out/lib \
+  --ruby_ads_out=/workspace/out/ \
+  --ruby_ads_opt="configuration=/workspace/config.yml" \
+  `find /workspace/in/ -name *.proto`
+
+# Fix file paths
+# Ensure google_ads exists
```

Comment:
Are you sure you want to expose the docker-specific paths? Do users care that the directory `/workspace/` is being used? And users won't see any output, as this command is silent AFAIK.

Reply:
@dazuma Thoughts on these suggestions?

Reply:
@blowmage You can take a few of these suggestions and omit others, or change them, or just print a single message saying that we're moving files to `google_ads`; I just wanted to suggest that we announce what's happening.

Reply:
I honestly don't think the output is valuable to users (even if it didn't leak the internal paths used within the container). I wouldn't mind seeing general "Now generating client files..." and "Now moving files to final locations..." or something like that, but this is all implementation detail. The protoc call itself doesn't produce any output either.

Reply:
I think if we do this for ads we should do it for the default and cloud generators as well. In that sense it might make more sense for that change to be a separate PR.

Reply:
Let's move to #166.